src/Sql/Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -29,11 +29,16 @@ public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatab
         /// Gets or sets backup retention days.
         /// </summary>
         [Parameter(Mandatory = true,
-            Position = 3,
             HelpMessage = "The backup retention setting, in days.")]
-        [ValidateNotNullOrEmpty]
-        public int RetentionDays{ get; set; }
+        public int? RetentionDays { get; set; }
 
+        /// <summary>
+        /// Gets or sets differential backup interval hours.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "The differential backup interval, in hours.")]
+        public int? DiffBackupIntervalInHours { get; set; }	
```

Comment:
(1/2) make the parameter optional, and do not use nullable types
This will make the parameter easier to undertand and use
```suggestion
        [Parameter(Mandatory = false,
            HelpMessage = "The differential backup interval, in hours.")]
        public int DiffBackupIntervalInHours { get; set; }	
```

Reply:
Modified.